### ISSUE TITLE
PMF Issue - Not getting the example results and Getting Error Instead

### ISSUE THREAD

**Issue by ImanB** [guoguibing/librec#37]
Hello,

It is a very nice Java Package which is very useful. Thank you. 
I have an issue. I am trying to run the package with the default dataset and just test it. I am trying to set the parameters as they are mentioned in the example page, but not only I don'g get same results, but also I get error. Here is my config file for PMF:

```
dataset.ratings.wins=.\\demo\\Datasets\\FilmTrust\\ratings.txt
dataset.ratings.lins=./demo/Datasets/FilmTrust/ratings.txt

ratings.setup=-columns 0 1 2 -threshold -1

recommender=PMF
evaluation.setup=cv -k 5 -p on --rand-seed 1 --test-view all 
item.ranking=off -topN -1 -ignore -1

num.factors=10
num.max.iter=1000

learn.rate=0.0030 -momentum 0.8 -moment 0.8 -bold-driver
reg.lambda=0.1 -u 0.1 -i 0.1 -b 0.1 -s 0.001

output.setup=on -dir ./demo/Results/

```

Here are the example configurations: 
factors=10, reg=0.1, learn.rate=30, momentum=0.8, max.iter=200

and here is what I get:

```
Dataset: ...demo/Datasets/FilmTrust/ratings.txt
With Specs: {Users, Items, Ratings} = {1508, 2071, 35497}, Scale = {0.5,1.0,1.5,2.0,2.5,3.0,3.5,4.0}
With Setup: cv -k 5 -p on --rand-seed 1 --test-view all
Fold [1]: training amount: 28395, test amount: 7099
PMF: [factors, lRate, maxLRate, regB, regU, regI, iters, boldDriver] = [10, 30.0, -1.0, 0.1, 0.1, 0.1, 200, true]
PMF fold [1] iter 1: loss = NaN, delta_loss = NaN, learn_rate = 30.0
Loss = NaN or Infinity: current settings does not fit the recommender! Change the settings and try again!
```

The point is that if I make the learning rate to some small number: 0.003, it won't give me error. Can anyone help me? I am using the latest version of the package.

**Comment by graytowne** [comment]
it seems like your learning rate(lRate) is too high, you can try value from 0.01 to 0.1
